.eslintrc.js

```diff
@@ -137,7 +137,7 @@ module.exports = {
     'no-path-concat': 'error',
     'no-proto': 'error',
     'no-redeclare': 'error',
-    'no-restricted-modules': ['error', 'sys'],
+    'no-restricted-modules': ['error'],
```

Comment:
```suggestion
    'no-restricted-modules': 'error',
```

Reply:
This line should be deleted entirely. If you don't specify a second argument, the rule does nothing.